serverlessworkflow/modules/ROOT/pages/core/understanding-workflow-error-handling.adoc

```diff
@@ -7,31 +7,43 @@ Understanding workflow error handling
+An link:{spec_doc_url}#error-definition[error definition] in Serverless Workflow is composed of a `name` and `code` parameters. The `name` is a short description of an error, such as `wrong parameter`, and the `code` parameter helps the implementation to identify the error.
+
+[NOTE]
+====
+In Kogito, the `code` parameter is mandatory, which consists of the fully qualified Java class name of an link:{java_exception_url}[exception type].
+====
 
-The steps used by Kogito to determine if an exception should be handled by an error definition are:
+To determine if an exception can be handled by an error definition, the Serverless Workflow first checks whether or not the type of the Java exception is an instance of the exception that is specified in the error definition. Otherwise, the Serverless Workflow recursively applies the same check to the root of the original Java exception till the exception chain is exhausted.
 
-* It checks if the type of the Java Exception being thrown is an instance (which includes any descendant) of the Java Exception specified in `code` property of the error definition. 
-* If this not the case, it recursively applies the same check to the root cause of the original Java Exception, till the exception chain is exhausted. 
+[[ref-example-error-handling]]
+== Example of error handling
 
-== Error handling example 
+The workflow in the link:{kogito_sw_examples_url}/serverless-workflow-error-quarkus[`serverless-workflow-error-quarkus`] example application describes the error handling feature as follows: 
 
-The workflow in the link:{kogito_sw_examples_url}/serverless-workflow-error-quarkus[Serverless Workflow Error Quarkus] example illustrates the error handling feature, as shown in the image below. 
+.Example of error handling
+image::core/error_handling.png[]
 
-image::core/error_handling.png []
+In the `serverless-workflow-error-quarkus` example application, the workflow accepts an integer number as an input model. When the workflow starts, it invokes the `isEven` Java service, which passes the the defined integer number as a parameter. The link:{kogito_sw_examples_url}serverless-workflow-error-quarkus/src/main/java/org/kie/kogito/examples/EvenService.java[`isEven` Java service] throws an `IllegalArgumentException` exception if the passed integer number is odd. This means that if the integer number is even, the Serverless Workflow transitions to the even state, however, if the integer number is odd, the Serverless Workflow transitions to the odd state, resulting in a different model output.
```

Comment:
```suggestion
In the `serverless-workflow-error-quarkus` example application, the workflow accepts an integer number as an input model. When the workflow starts, it invokes the `isEven` Java service, which passes the defined integer number as a parameter. The link:{kogito_sw_examples_url}serverless-workflow-error-quarkus/src/main/java/org/kie/kogito/examples/EvenService.java[`isEven` Java service] throws an `IllegalArgumentException` exception if the passed integer number is odd. This means that if the integer number is even, the Serverless Workflow transitions to the `even` state, however, if the integer number is odd, the Serverless Workflow transitions to the `odd` state, resulting in a different model output.
```

Small change just to highlight that even and odd, in this context, are state names.

Reply:
I already changed